melodies_monet/driver.py

```diff
@@ -898,7 +944,24 @@ def plotting(self):
                                 text_dict=text_dict,
                                 debug=self.debug
                             )
+                            if cal_reg:
+                                outname = "{}.{}".format(outname, p_label)
+                                splots.make_spatial_exceedance(
```

Comment:
Do you want to make the spatial exceedences plot, a plot_type instead? You could specify in the yaml file and readthedocs that the plot is only for regulatory calculations and add an if statement to only make the plot for regulatory calculations. This would make things more consistent, but we can also update this later if you prefer.